3DAmsterdam/Assets/Netherlands3D/Scripts/Selection/GridSelection/GridSelection.cs

```diff
@@ -272,6 +272,10 @@ private void MoveSelectionBlock()
 
 		public void RenderGridToThumbnail()
 		{
+			
+			//TODO: send this boundingbox to the mesh selection logic, and draw the sidepanel
+			PropertiesPanel.Instance.OpenObjectInformation("Grid selectie", true, 10);
+
```

Comment:
Is this the right place for this? 🤔 Feels a bit strange that the sidepanel is opened from our GridSelection which job it only is to do a grid selection. The different usecases would probably want to open the sidepanel with a different tab/title

Reply:
This was when I was trying to make the Grid multipurpose without delving too much into the details of how it works. This method is solely called from the Download script. I kept it on the grid so that the grid scaleblock can be hidden, render the image and enabled again. Should I try to keep that functionality? Should the grid be even visible in the rendered thumbnail?